src/scene/container/bounds/getLocalBounds.ts

```diff
@@ -46,42 +40,41 @@ function _getLocalBounds(target: Container, bounds: Bounds, parentTransform: Mat
 {
     if (!target.visible || !target.measurable) return;
 
-    // // make sure localTransform is upto date...
     if (target.didChange)
     {
         updateLocalTransform(target.localTransform, target);
     }
 
     const localTransform = target.localTransform;
 
-    const relativeTransform = Matrix.shared.appendFrom(localTransform, parentTransform).clone();
+    const relativeTransform = getMatrix().appendFrom(localTransform, parentTransform);
```

Comment:
Assuming you'll probably want to merge [this guy](https://github.com/pixijs/pixijs/pull/9972#pullrequestreview-1781281977) first (with the pool API changes), then rebase this branch and update the pool API usage here?

Reply:
cool! updated :D